utils/entrypoint.sh

```diff
@@ -318,7 +318,7 @@ initialize
 if [ -z "$TZ" ]; then
     TZ="UTC"
 fi
-echo "date.timezone = $TZ" >> $PHPINI
+echo "date.timezone = $TZ" >> ${PHPINI:-/etc/php.ini}
```

Comment:
This should not be necessary.

Reply:
Not having this set here throws a warning when running this script, this is because $PHPINI is declared & assigned within the `initialize` function only. Either it should be declared in the main body of the program or set to some default value (as shown here).

Reply:
Variables in bash are global by default, even when declared in a function. Can you show the warning you are referring to? Since this affects all the variables defined in the initialize function, I'd rather fix this properly than just assign default at this point in the script.

Reply:
If it pops up again, maybe we can use `declare` or `export` for each variable in the initialize subroutine.